views/partials/_subnav.njk

```diff
@@ -10,6 +10,13 @@
           {% for subitem in items %}
             <li class="app-subnav__section-item{% if subitem.url == path %} app-subnav__section-item--current{% endif %}">
               <a class="govuk-link" href="/{{ subitem.url }}">{{ subitem.label }}</a>
+              {% if subitem.url == path %}
+               {% for link in subitem.links %}
+                 {% if link.depth == 2 %}
+                   <a href="/{{ subitem.url }}#{{ link.url }}" class="app-subnav__sub-item">{{ link.text }}</a>
```

Comment:
I think for accessibility reasons, we need to consider the semantics of this.

It may need to be a nested list (if we think we need to have two navigations...)